packages/sync/legacy/class.jetpack-sync-actions.php

```diff
@@ -29,7 +30,7 @@ static function init() {
 		add_action( 'wp_cron_importer_hook', array( __CLASS__, 'set_is_importing_true' ), 1 );
 
 		// Sync connected user role changes to .com
-		Jetpack_Sync_Users::init();
+		new Jetpack_Sync_Users( $connection );
```

Comment:
As per comment on the constructor below, we could remove $connection from the constructor and just wrap this in a condition that says `if $conn->is_active() { new Jetpack_Sync_Users(); }`

Reply:
Nice 👍